src/main/webapp/WEB-INF/jsp/profile.jsp

```diff
@@ -29,6 +29,65 @@ label {
 </head>
 
 <body>
+
+	<center>
```

Comment:
it shouldn't be allowed.
I have done initially like that but someone modified the jsp .so it has
changed that way.

On Thu, Oct 22, 2015 at 12:40 PM, Nagarjuna Myla <notifications@github.com>
wrote:

> In src/main/webapp/WEB-INF/jsp/profile.jsp
> <https://github.com/chasethenag420/sbs/pull/87#discussion_r42796006>:
>
> > @@ -29,6 +29,65 @@ label {
> >  </head>
> >
> >  <body>
> > +
> > +	<center>
>
> So does modify profile isn't allowed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chasethenag420/sbs/pull/87/files#r42796006>.
>